packages/notification-service/src/contracts/Erc20.ts

```diff
@@ -0,0 +1,312 @@
+export const ERC_20 = {
```

Comment:
We can get this abis, directly from the package `@fuel-bridge/solidity-contracts` or `@fuel-brigde/fungible-tokens` take a look on the PR I replaced it here: https://github.com/FuelLabs/fuels-portal/blob/0bc8c0f4513c8b964e904a2ccc27bffec93c5794/packages/app/src/systems/Chains/eth/contracts/Erc20.ts#L1